commands.go

```diff
@@ -306,14 +310,14 @@ func newQemuHelper(c Command) qemuHelper {
 	case "amd64", "i386":
 		/* Dummy, no qemu */
 	default:
-		log.Panicf("Don't know qemu for Architecture %s", c.Architecture)
+		return q, fmt.Errorf("Don't know qemu for architecture %s", c.Architecture)
```

Comment:
Should probably be `return nil, err...` aka `s/q/nil/`

Reply:
it should indeed

Reply:
sorted that one out